setup.py

```diff
@@ -27,15 +27,7 @@
     namespace_packages=[],
     include_package_data=True,
     zip_safe=True,
-    install_requires=[
-        "banal >= 1.0.1, <2.0.0",
-        "normality >= 2.1.1, <3.0.0",
-        "fakeredis == 1.10.1",
-        "sqlalchemy >= 1.3",
-        "structlog >= 20.2.0, < 23.0.0",
-        "colorama >= 0.4.4, < 1.0.0",
-        "pika == 1.3.1",
-    ],
+    install_requires=[],
```

Comment:
I still think it's good to have this section around. Considering [what the docs say about requirements.txt vs install_requires](https://packaging.python.org/en/latest/discussions/install-requires-vs-requirements/#), we would have this as loose recommendations as supposed to the pinned versions in `requirements.txt`. What do you think?

Here's what the updated values could look like:
```
banal >= 1.0.6, < 2.0.0
normality >= 2.4.0, < 3.0.0
fakeredis == 1.10.1, < 2.0.0
sqlalchemy >= 2.0.4, < 3.0.0
structlog >= 20.2.0, < 23.0.0
colorama >= 0.4.4, < 1.0.0
pika >= 1.3.1, < 2.0.0
```